shared/common-adapters/with-tooltip.desktop.js

```diff
@@ -22,6 +24,16 @@ class WithTooltip extends React.Component<Props, State> {
     this.setState({mouseIn: false})
   }
   _setAttachmentRef = attachmentRef => (this._attachmentRef = attachmentRef)
+  componentDidUpdate(prevProps: Props, prevState: State) {
+    if (!prevState.mouseIn && this.state.mouseIn) {
+      // Set visible after Toast is mounted, to trigger transition on opacity.
+      // Note that we aren't doing anything to make ease out work. We don't
+      // keep Toast mounted all the time (which would make this unnecessary)
+      // because in that case the position of the overlay can be messed up when
+      // not visible, causing ghost unclickable areas.
+      this.setState({visible: true})
+    }
```

Comment:
Can we set `visible` to false in the opposite condition? I think not doing that is causing the fade-in to only happen once per mount

Reply:
good catch!